packages/mdc-rtl/README.md

```diff
@@ -28,76 +26,20 @@ npm install --save @material/rtl
 
 ## Usage
 
-Simply `@import "@material/rtl/mixins";` and start using the mixins. Each mixin is described below.
+### Sass Mixins
 
-### mdc-rtl
+`$base-property` is a base box-model property. e.g. margin / border / padding.
```

Comment:
Listing the args out here looks a bit unnatural to me. Can we put it as part of the following table and use a more explanatory name, such as `position-property`?